api/src/test/java/org/openmrs/module/dhisreport/api/adx/AdxTypeTest.java

```
@@ -0,0 +1,111 @@
+package org.openmrs.module.dhisreport.api.adx;
+
+import static org.junit.Assert.*;
+
+import java.io.ByteArrayInputStream;
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+import java.io.StringWriter;
+import java.io.Writer;
+import java.math.BigDecimal;
+import java.util.GregorianCalendar;
+import java.util.List;
+import java.util.Map;
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Marshaller;
+import javax.xml.bind.Unmarshaller;
+import javax.xml.datatype.DatatypeConfigurationException;
+import javax.xml.datatype.DatatypeFactory;
+import javax.xml.datatype.XMLGregorianCalendar;
+import javax.xml.namespace.QName;
+import javax.xml.xpath.XPathExpressionException;
+import org.junit.Test;
+
+public class AdxTypeTest {
+
+	@Test
+	public void test() throws JAXBException, FileNotFoundException, XPathExpressionException {
+		ObjectFactory of = new ObjectFactory();
+		AdxType adxt = of.createAdxType();
+		Map<QName, String> hm2 = adxt.getOtherAttributes();
+		QName qn3 = new QName("adxattri");
+		hm2.put(qn3, "adx");
+		XMLGregorianCalendar date3 = null;
+		try {
+			date3 = DatatypeFactory.newInstance().newXMLGregorianCalendar(new GregorianCalendar(2016, 06, 14));
+		} catch (DatatypeConfigurationException e) {
+			e.printStackTrace();
```

Don't you just wanna let this bubble through?